ovm-ironic/ovm_ironic/manager/manager_ironic.py

```diff
@@ -60,7 +66,12 @@ def _get_ironic_client(self):
             'os_password': self.__password,
             'os_auth_url': self.__auth_url,
             'os_tenant_name': self.__tenant_name,
-            'os_ironic_api_version': self.__api_version
+            'os_ironic_api_version': self.__api_version,
+            'os_project_name': self.__project_name,
+            'os_region_name': self.__region_name,
+            'os_user_domain_id': self.__user_domain_id,
+            'os_project_domain_id': self.__project_domain_id,
+            'ironic_url': self.__ironic_url
```

Comment:
May you send a patch with os_project_id, os_user_domain_name and os_projetc_domain_name too?

They only have to be forwarded too the ironic client, it figures out how to deal with possible conflicts. It would improve a lot the usability.